extensions/atomic/package.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "atomic",
+  "title": "Atomic Data",
+  "description": "Search inside your Atomic-Server data",
+  "icon": "command-icon.png",
+  "author": "joepio",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Atomic Search",
+      "subtitle": "Find things stored in your Atomic Server",
```

```suggestion
      "title": "Search Data",
      "subtitle": "Atomic Server",
```
Subtitle is used for the name of the `service` – could be changed to something like this